crates/controller/src/mouse.rs

```diff
@@ -218,6 +237,13 @@ fn update_buttons(
                     match drag_resolution {
                         DragResolution::Point(_) => {
                             clicks.send(MouseClicked::new(event.button));
```

Comment:
I would send this event only if double click was not triggered so the two events are never send at once. I have no strong opinion though.

Reply:
In js, a second click event is sent followed by a double click event.

Reply:
I didn't know that. Better to stick to conventions so lets keep it as is.